lightstep_incident_response/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b61db30e-3cbc-4dbe-b055-824f9e46e006",
+  "app_id": "lightstep_incident_response",
```

Comment:
Looks like our validator requires the `app_id` to not use `_`, could you change this to `lightstep-incident-response` instead?

Reply:
That should fix the validations CI job.

Reply:
replaced the underscore with hyphen